templates/zerver/app/index.html

```diff
@@ -147,6 +147,7 @@
 <div id="scheduled_messages_overlay_container"></div>
 <div id="settings_overlay_container" class="overlay" data-overlay="settings" aria-hidden="true">
 </div>
+<div id="message_edit_history_overlay_container"></div>
```

Comment:
Let's go with our preferred [hyphenated style](https://zulip.readthedocs.io/en/latest/subsystems/html-css.html#use-clear-unique-names-for-classes-and-object-ids) here and throughout, so `message-edit-history-overlay-container`. Fix up any other classes and IDs that include underscores.

Reply:
sure, can we change classes and IDS that have underscores and are not related to the `message-edit-histoy` in the new commit/PR?

Reply:
I think it's okay to just focus on the new/modified and related ones here. It's not so high priority that unrelated ones need to be fixed...my preference is to just make sure we're doing this kind of work in PRs that touch classes/IDs with this different style. Just be sure to use `git grep` to make sure you're catching all instances of the underscored versions that you're trying to fix.